template/gitignore

```diff
@@ -0,0 +1,7 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
It can't be named .gitignore due to npm/npm#1862, which is why we need to move it after the fact.